src/TestState.elm

```diff
@@ -1,5 +1,6 @@
-module TestState exposing (TestState, advanceTime, drain, queueEffect, routeChangeHelper, simulateLoadUrlHelper, update, withSimulation)
+module TestState exposing (TestState, advanceTime, drain, queueEffect, simulateLoadUrlHelper, update, urlChangeHelper, urlRequestHelper, withSimulation)
```

Comment:
`simulateLoadUrlHelper` can be made private now, I think

Reply:
Done.